plugin.json

```diff
@@ -91,4 +91,4 @@
             }
         ]
     }
-}
```

Comment:
@dimashasbi this should be the same as master. Can you revert this as this is not a related change

Reply:
on this commit [2f6fdcad](2f6fdcad4dace4a27e7e59fe46c58729d9019945) is already equal to master

Reply:
fixed here 85c440123cdac0b74a6ee6edcb6ef8ecc80ba37b